scss/_progress.scss

```diff
@@ -1,32 +1,37 @@
+// Progress animations
 @keyframes progress-bar-stripes {
   from { background-position: $progress-height 0; }
   to { background-position: 0 0; }
 }
 
+// Basic progress bar
 .progress {
   display: flex;
+  align-items: center;
+  height: $progress-height;
   overflow: hidden; // force rounded corners by cropping it
   font-size: $progress-font-size;
   line-height: $progress-height;
```

Comment:
ah, it looks like align-items is indeed safe to remove, but the height is needed (empty bars will collapse to zero) with no explicit height.